test/e2e/cloud/gcp/resize_nodes.go

```diff
@@ -54,7 +54,7 @@ var _ = SIGDescribe("Nodes [Disruptive]", func() {
 		systemPods, err := e2epod.GetPodsInNamespace(c, ns, map[string]string{})
 		framework.ExpectNoError(err)
 		systemPodsNo = int32(len(systemPods))
-		if strings.Index(framework.TestContext.CloudConfig.NodeInstanceGroup, ",") >= 0 {
```

Comment:
staticcheck show that we neet to change this and   test/cloud not in  hack/.staticchek_failures